src/main/java/ca/xqz/tweetmouth/Tweet.java

```diff
@@ -23,19 +39,15 @@
 
     public static Tweet getTweet(List<String> tokens) {
         if (tokens.size() != Tweet.NUM_NONDERIVED_FIELDS) {
-            // TODO: Replace with logging
```

Comment:
Why? Shouldn't we be logging this stuff?
Adding a TODO comment is good enough for me at this point though.